packages/core/contracts/common/implementation/AncillaryData.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity ^0.8.0;
  * @title Library for encoding and decoding ancillary data for DVM price requests.
  * @notice  We assume that on-chain ancillary data can be formatted directly from bytes to utf8 encoding via
  * web3.utils.hexToUtf8, and that clients will parse the utf8-encoded ancillary data as a comma-delimitted key-value
- * dictionary. Therefore, this libraries provides internal methods that aid appending to ancillary data from Solidity
+ * dictionary. Therefore, this library provides internal methods that aid appending to ancillary data from Solidity
```

Comment:
- In `AncillaryData.sol` on [line 8](https://github.com/UMAprotocol/protocol/blob/a3bf46270787cbaae4ed2218f064b1217c153a50/packages/core/contracts/common/implementation/AncillaryData.sol#L8) "libraries" should be "library".